image/gif.ksy

```diff
@@ -34,7 +34,8 @@ seq:
     doc-ref: https://www.w3.org/Graphics/GIF/spec-gif89a.txt - section 18
   - id: blocks
     type: block
-    repeat: eos
+    repeat: until
+    repeat-until: _.block_type.to_i == 0x3b
```

Comment:
Um, why don't we just use

    repeat-until: _.block.type == block_types::end_of_file

or, better yet, something like:

    repeat-until: _io.eof or _.block.type == block_types::end_of_file

... to make sure we don't fail on truncated files which lack EOF block?

Reply:
Due to my inexperience with ksy :sweat_smile: 
I'll push the new changes :+1: